certification/internal/engine/errors.go

```diff
@@ -0,0 +1,20 @@
+package engine
```

Comment:
Are the comments in this package still valid? ie didn't the term `Engine` go away for everything but our own engine?